### PR TITLE
Add Khang Huu Nguyen profile

### PR DESCRIPTION
**Pull request by kn107**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -378,4 +378,17 @@ export const mentorshipTeam = [
 			},
 		],
 	},
+	{
+    name: 'Khang Huu Nguyen',
+    title: 'Student @ UMass Amherst',
```
Comment (khoa165) [inline index.js]: Let's use full university name

File: index.js [hunk 2]
```diff
@@ -378,4 +378,17 @@ export const mentorshipTeam = [
 			},
 		],
 	},
+	{
+    name: 'Khang Huu Nguyen',
```
Comment (khoa165) [inline index.js]: hmmm not sure why but it looks like the code is not indented as properly as the above

File: index.js [hunk 3]
```diff
@@ -378,4 +378,17 @@ export const mentorshipTeam = [
...
+    title: 'Student @ UMass Amherst',
+    hobbies: 'J-pop, Gaming, Anime',
```
Comment (khoa165) [inline index.js]: can we use lowercase for `gaming` and `anime` to be consistent with others above

File: index.js [hunk 4]
```diff
@@ -378,4 +378,17 @@ export const mentorshipTeam = [
...
+    avatar: 'https://media.licdn.com/dms/image/D4E03AQFvJxo3u961kQ/profile-displayphoto-shrink_400_400/0/1708669047854?e=1716422400&v=beta&t=EyfNJkgkzmLMgQv9iAOQgKNBffPSMjFob3Q2lnHU2UE',
```
Comment (khoa165) [inline index.js]: replace with this `https://res.cloudinary.com/khoa165/image/upload/v1711089916/viettech/huukhang.jpg`

**Review by khoa165 — COMMENTED** [review]
Left some feedback

File: index.js (hunk 2 above)
Comment (kn107) [reply index.js]: I think it's because I'm using space and the original code is using tabs

**Comment by kn107** [comment]
I've reformatted based on your comments and changed the indentation to tabs to stay consistent with the original styling

**Review by khoa165 — APPROVED** [review]
LGTM. Thanks for making the changes

**Pull request merged**